langlib/langlib-test/src/test/resources/test-src/decimallib_test.bal

```diff
@@ -45,6 +45,18 @@ function testRound(decimal arg) returns decimal {
     return decimals:round(arg);
 }
 
```

Comment:
Shall we also add tests with named arguments to round? That way we test the parameter names also.

Reply:
Lets address them in https://github.com/ballerina-platform/ballerina-lang/issues/35756